beeai/README.md

```diff
@@ -10,6 +10,138 @@ Three agents process tasks through Redis queues:
 - **Rebase Agent**: Updates packages to newer upstream versions
 - **Backport Agent**: Applies specific fixes/patches to packages
 
+### Entry Criteria
+
+Issues enter the system automatically via the Jira Issue Fetcher when they meet these criteria:
+- Belongs to the `Jotnar_1000_packages` filter
+- Status is "New"
+- No `jotnar_*` labels other than `jotnar_retry_needed`
+
+### Queue Transitions and Label Management
+
+The system uses Redis queues to route issues between agents and applies JIRA labels to track progress:
+
+**From Triage Agent:**
+- **REBASE decision** → `rebase_queue` + applies `jotnar_in_progress` label
+- **BACKPORT decision** → `backport_queue` + applies `jotnar_in_progress` label
+- **CLARIFICATION_NEEDED decision** → `clarification_needed_queue` + applies `jotnar_needs_attention` label
+- **NO_ACTION decision** → `no_action_list` + applies `jotnar_no_action_needed` label (or `jotnar_cant_do` when the system is not capable of solving the problem)
+- **ERROR/Exception** → `error_list` + applies `jotnar_errored` label
+
+**From Rebase Agent:**
+- **Success** → `completed_rebase_list` + applies `jotnar_rebased` label
+- **Failure** → `error_list` + applies `jotnar_errored` label
+- **Retry** → remains in `rebase_queue` (keeps `jotnar_in_progress` label)
+
+**From Backport Agent:**
+- **Success** → `completed_backport_list` + applies `jotnar_backported` label
+- **Failure** → `error_list` + applies `jotnar_errored` label
+- **Retry** → remains in `backport_queue` (keeps `jotnar_in_progress` label)
+
+### Re-triggering Mechanism
+
+Issues can be re-triggered through the workflow in two ways:
+1. **Remove any existing `jotnar_*` label** - allows the issue to re-enter the system on the next fetcher run
+2. **Add the `jotnar_retry_needed` label** - intended for package maintainers who have made changes (e.g., updated issue description) and want to retrigger the workflow
+
+### Maintainer Review Process
+
+The Jotnar team can require maintainer review by applying the `jotnar_needs_maintainer_review` label to an issue. This is especially important for FuSa (Functional Safety) project packages.
+
+The `jotnar_fusa` label will be automatically added by the triage agent to JIRA issues involving FuSa packages, and related merge requests will need to be reviewed and handled by subject matter experts.
+
+### Workflow Diagram
```

Comment:
do you think we could add this directly to the `Queue Transitions and Label Management` section, as it captures that mostly?

Reply:
I am also thinking if we should omit some of the details, so that this doesn't get outdated very easily, wdyt?

Reply:
> do you think we could add this directly to the `Queue Transitions and Label Management` section, as it captures that mostly?

yes, that is what I wanted 😅 good catch!